### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/18.1/k8s1_%E6%A6%82%E5%BF%B5%E5%92%8C%E8%BF%90%E8%A1%8C%E6%9C%BA%E5%88%B6%E7%AE%80%E4%BB%8B

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#52] [General]
# https://www.xiaogenban1993.com/blog/18.1/k8s1_%E6%A6%82%E5%BF%B5%E5%92%8C%E8%BF%90%E8%A1%8C%E6%9C%BA%E5%88%B6%E7%AE%80%E4%BB%8B

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/18.1/k8s1_%E6%A6%82%E5%BF%B5%E5%92%8C%E8%BF%90%E8%A1%8C%E6%9C%BA%E5%88%B6%E7%AE%80%E4%BB%8B

**Comment by sunwu51** [comment]
这篇的tags好像掉了。。
